Makefile

```diff
@@ -29,6 +29,12 @@ ifndef TEST_FLAKEY
 endif
 CARGO_TEST = $(CARGO) test --frozen $(RELEASE) $(TEST_FLAGS)
 
+DOCKER = docker
+DOCKER_BUILD = docker build
+ifdef DOCKER_TAG
+	DOCKER_BUILD = docker build -t $(DOCKER_TAG)
```

Comment:
Previously, in the `conduit` repository, there was a `bin/root-tag` script that could be used to determine what the tag would be if it's not overridden. It might be nice if the makefile had similar functionality eventually, but that may be out of scope for this change...